tests/test_package_client.py

```diff
@@ -36,7 +47,30 @@ def test_install_package(self, setup_tester_ssh_connection):
         result = setup_tester_ssh_connection.run('mender -version', hide=True)
         assert Helpers.mender_version in result.stdout
 
-        #TODO: check for inventory scripts, update modules, ...
+        # Check installed files
+        setup_tester_ssh_connection.run('test -x /usr/bin/mender', hide=True)
+        setup_tester_ssh_connection.run('test -d /usr/share/mender/modules/v3', hide=True)
+        for module in self.expected_update_modules:
+            module_path = os.path.join("/usr/share/mender/modules/v3", module)
+            setup_tester_ssh_connection.run('test -x {mod}'.format(mod=module_path), hide=True)
+        setup_tester_ssh_connection.run('test -d /usr/share/mender/modules/v3', hide=True)
+        for inventory in self.expected_inventory_files:
+            inventory_path = os.path.join("/usr/share/mender/inventory", inventory)
+            setup_tester_ssh_connection.run('test -x {invent}'.format(invent=inventory_path), hide=True)
+        for identity in self.expected_indentity_files:
+            identity_path = os.path.join("/usr/share/mender/identity", identity)
+            setup_tester_ssh_connection.run('test -x {ident}'.format(ident=identity_path), hide=True)
+        setup_tester_ssh_connection.run('test -d /etc/mender', hide=True)
+        setup_tester_ssh_connection.run('test -f /etc/mender/mender.conf', hide=True)
+        setup_tester_ssh_connection.run('test -f /etc/mender/mender.conf.demo', hide=True)
+        setup_tester_ssh_connection.run('test -f /etc/mender/mender.conf.production', hide=True)
+        setup_tester_ssh_connection.run('cmp /etc/mender/mender.conf.production /etc/mender/mender.conf', hide=True)
+        setup_tester_ssh_connection.run('test -f /lib/systemd/system/mender.service', hide=True)
```

Comment:
Tip: I think you can do:

```
with settings(hide=True):
    setup_tester_ssh_connection.run('test -x /usr/bin/mender')
```

to avoid having to use `hide=True` everywhere.

Reply:
Great! Thanks

Reply:
So the `with settings...` is a deprecated feature from Fabric 1 to Fabric 2, see http://www.fabfile.org/upgrading.html#configuration

I added the hide=True to the connection config, so that we nevertheless avoid repeating it in every `run` call.